qcodes/instrument_drivers/tektronix/Keithley_2400.py

```diff
@@ -97,6 +98,7 @@ def __init__(self, name, address, **kwargs):
                            docstring="Measure resistance from current and voltage "
                                      "Note that it is an error to read current "
                                      "and voltage with output off")
+        self.write(':TRIG:COUN 1;:FORM:ELEM VOLT,CURR')
```

Comment:
Is this the same mode that reset would put the instrument in? Could you document (for instance in a comment) what this does

Reply:
Comment added in the driver itself, and pushed in a new commit. Did you mean to comment just here instead? I didn't realise there was a github comment feature!

Reply:
Thanks

Reply:
both these commands aught to be represented as qcodes parameters (or at least methods), in which case this explanation wouldn't be needed. but we can leave this to another PR.